src/Operation/Aggregate.php

```diff
@@ -297,6 +302,14 @@ private function createCommand(Server $server, $isCursorSupported)
             $cmd['hint'] = is_array($this->options['hint']) ? (object) $this->options['hint'] : $this->options['hint'];
         }
 
+        if (isset($this->options['maxAwaitTimeMS'])) {
+            $cmdOptions['maxAwaitTimeMS'] = $this->options['maxAwaitTimeMS'];
+        }
+
+        if (isset($this->options['maxTimeMS'])) {
```

Comment:
Is this necessary? I think it's already covered by `foreach (['comment', 'maxTimeMS'] as $option)` about 15 lines prior.